daemon/oci_linux.go

```diff
@@ -988,6 +986,10 @@ func WithResources(c *container.Container) coci.SpecOpts {
 			},
 			Pids: getPidsLimit(r),
 		}
+		if r.BlkioWeight != 0 {
+			w := r.BlkioWeight
+			specResources.BlockIO.Weight = &w
+		}
```

Comment:
Definitely not a blocker, but would it make sense to have a `getBlkioWeight()` func (like we do for the other options above?